cmd/render.go

```diff
@@ -42,6 +47,7 @@ provided, Qri will render the dataset with a default template.`,
 	}
 
 	cmd.Flags().StringVarP(&o.Template, "template", "t", "", "path to template file")
+	cmd.Flags().BoolVarP(&o.UseReadme, "readme", "r", false, "whether to use the readme component")
```

Comment:
let's make the readme the default, and instead flag on weather to use the viz component. Readme's are _definitely_ going to get used, viz might get deprecated